test/performance/observed-concurrency/reporter.lua

```diff
@@ -14,7 +14,16 @@
 
 -- parses a response as returned by the system under test
 function parse_response(res)
+  if (res == nil or res == '') then
+    print('ERROR: Empty response from the test.')
+    os.exit()
+  end
   local split_at, _ = res:find(',', 0)
+  if (split_at == nil or split_at == '') then
+    print('ERROR: Split_at is nil.')
+    print(res)
```

Comment:
This `print()` is not useful, don't you agree? `res` is `nil` or empty, it's already stated by the error message.

Reply:
res is nil|| empty is checked before this. This is when res ia an error, so split_at will be nil or empty. Updated the message to reflect that.